### PR TITLE
#616 Finish entry point updates by updating generator

### PR DESCRIPTION
**Pull request by Westbrook**
- update deps and config for rollup
- update config for webpack

I meant to get this into the last PR for this issue. Working on the right communication practices as I get up to speed on the repo.

### CODE REVIEW

**Comment by Westbrook** [comment]
This should be good to go if anyone’s ready for a review!

File: packages/create/src/generators/building-webpack/templates/static/webpack.config.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 const path = require('path');
-const createDefaultConfig = require('@open-wc/building-webpack/modern-config.js');
+const { createDefaultConfig } = require('@open-wc/building-webpack');
```
Comment (LarsDenBakker) [inline packages/create/src/generators/building-webpack/templates/static/webpack.config.js]: does webpack need a version bump too?

**Comment by LarsDenBakker** [comment]
We can squash this to 1 commit on merge, right?

**Comment by Westbrook** [comment]
Yes, I _love_ squash.

Webpack is post v1.0.0 so when we make feature releases, it will just take them…yay! 🌮 Happy to make a manual change if you’d prefer to be explicit: https://github.com/open-wc/open-wc/blob/master/packages/create/src/generators/building-webpack/templates/_package.json#L7

Discussed separately doing a v1.0.0 party on the features that we have that are ready for that on Slack, but I’m not 100% which ones are to push that initiative forward.

**Comment by LarsDenBakker** [comment]
Ah, yes you're right. We can keep it as is. Happy to bump everything to 1.x.y, I'm not a huge fan of 0.x.y releases unless we're at a very early stage.

**Pull request merged**